liquibase-core/pom.xml

```diff
@@ -108,7 +114,7 @@
                         </filter>
                     </filters>
                     <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
-                    <createDependencyReducedPom>true</createDependencyReducedPom>
+                    <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
I don't know enough about this change, but it does concern me that it could be a breaking change. I think it's worth discussing, or removing if not needed.

Reply:
@filipelautert do you think this will have any impact on any of the jars and poms we publish to central, github, etc?

Reply:
I set it back to `true`. It was some testing.